scripts/ci/upload-universe.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+const exec = require("child_process").exec;
+const fs = require("fs");
+
+if (process.argv.length < 3) {
+  throw new Error("You need to pass the version in as a command line argument");
+}
+function execute(command) {
+  return new Promise((resolve, reject) =>
+    exec(command, function(error, stdout, stderr) {
+      if (error) {
+        reject(stderr);
+      } else {
+        resolve(stdout);
+      }
+    })
+  );
+}
+
+const version = process.argv[2];
+const promise = fs.existsSync("../dcos-commons")
+  ? Promise.resolve()
+  : execute(
+      "git clone https://github.com/mesosphere/dcos-commons.git ../dcos-commons"
+    );
+
+promise
+  .then(log => {
+    console.log(log);
+    fs.writeFileSync(
+      `./release-${version}.tar.gz`,
+      fs.readFileSync("./release.tar.gz")
+    );
+
+    return execute(
+      `S3_BUCKET='dcos-ui-universe' S3_DIR_PATH='oss' S3_DIR_NAME='latest' ../dcos-commons/tools/build_package.sh 'dcos-ui' ./ -a "./release-${version}.tar.gz" aws ${version}`
```

Comment:
can't we just use `env` option? https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback